pom.xml

```diff
@@ -1,83 +1,128 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
     <groupId>com.github.dockerunit</groupId>
     <artifactId>dockerunit-parent</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>0.1.0-SNAPSHOT</version>
+    <relativePath/>
   </parent>
 
   <artifactId>dockerunit-consul</artifactId>
-  <version>2.0.0-SNAPSHOT</version>
 
-  <name>dockerunit-consul</name>
   <description>Java Framework for testing of dockerised applications and services. Consul based service discovery package.</description>
   <url>https://github.com/dockerunit/dockerunit-consul</url>
-  
-  <developers>
-    <developer>
-      <name>Quirino Zagarese</name>
-    </developer>
-    <developer>
-      <name>Pedro Nuno Santos</name>
-    </developer>
-    <developer>
-      <name>Vincenzo Candela</name>
-    </developer>
-    <developer>
-      <name>Francesco Sorice</name>
-    </developer>
-  </developers>
+
+  <scm>
+    <connection>scm:https://github.com/dockerunit/dockerunit-consul.git</connection>
+    <developerConnection>scm:https://github.com/dockerunit/dockerunit-consul.git</developerConnection>
+    <url>https://github.com/dockerunit/dockerunit-consul.git</url>
+  </scm>
+
   <properties>
-    <dockerunit.core.version>2.0.0-SNAPSHOT</dockerunit.core.version>
-    <lombok.version>1.16.6</lombok.version>
+    <lombok.version>1.18.8</lombok.version>
+    <jackson.version>2.9.8</jackson.version>
```

Comment:
You can bump to `2.9.9` if you want :+1:

Reply:
I'll leave it for dependabot